docs/getting_started/howto/adaptive_localization.ipynb

```diff
@@ -196,8 +195,10 @@
     "\n",
     "Be aware that adaptive localization comes with significant computational costs, which depend on two factors:\n",
     "\n",
-    "    - The number of parameters: More parameters lead to longer run times.\n",
-    "    - The correlation threshold: A higher threshold reduces computational time.\n"
+    "- The number of parameters: More parameters lead to longer run times.\n",
+    "- The correlation threshold: A higher threshold reduces computational time. \\\n",
+    "    Increasing the threshold makes it less likely that a parameter and a response are considered significantly correlated. \\\n",
```

Comment:
Is "\\\n" a typo or extra backslashes needed here?

Reply:
It's needed to add space in markdown. It basically looks better when you render the notebook.